classes-metodos-atributos/exercicio-03/src/main/java/MyProgram.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+
+public class MyProgram {
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
```

Comment:
O mesmo acontece nesse caso. É exibido um warning para verificar se é preciso fechar ou não o in. Verifique seus programas para que não vá nenhum warning ou erro para o branch principal.

Reply:
Não precisa fechar o "in", está correto a chamada da função Scanner in = new Scanner (System.in). O java executa este programa perfeitamente.

Reply:
Essa é a mensagem que está na aba PROBLEMAS/PROBLEMS do VSCode: Resource leak: 'in' is never closed.
Então, sim, precisa ser fechada e o código precisa vir sem nenhum PROBLEMA relatado nessa aba. 
Faça as verificações antes de enviar.

Reply:
Para solucionar o erro do in, na função Scanner, insira in.close();